mods_cocina_mappings/mods_to_cocina_note.txt

```diff
@@ -0,0 +1,65 @@
+1. Simple note
+<note>This is a note.</note>
+{
+  "note": [
+    {
+      "value": "This is a note."
+    }
+  ]
+}
+
+2. Note with type
+<note type="preferred citation">This is the preferred citation.</note>
+{
+  "note": [
+    {
+      "value": "This is the preferred citation.",
+      "type": "preferred citation"
+    }
+  ]
+}
+
+3. Multilingual note
+<note lang="eng">This is a note.</note>
+<note lang="fre">C'est une note.</note.
+{
+  "note": [
+    {
+      "parallelValue": [

```

Comment:
If we're looking at MODS, how do we know this is a parallel note or two different notes that happen to each be in different languages?

Reply:
Both would have the altRepGroup attribute set to the same value. So if two note elements both have altRepGroup=1, that signifies that they're parallel.

Reply:
Should that attribute be added on line 23-24 then?

Reply:
You're right - will update.